scene/gui/tab_bar.cpp

```diff
@@ -1305,6 +1331,27 @@ int TabBar::get_tab_width(int p_idx) const {
 	return x;
 }
 
+Size2 TabBar::_get_tab_icon_size(int p_index) const {
+	ERR_FAIL_INDEX_V(p_index, tabs.size(), Size2());
+	const TabBar::Tab &tab = tabs[p_index];
+	Size2 icon_size = tab.icon->get_size();
+
+	int icon_max_width = 0;
+	if (theme_cache.icon_max_width > 0) {
+		icon_max_width = theme_cache.icon_max_width;
+	}
+	if (tab.icon_max_width > 0 && tab.icon_max_width < icon_max_width) {
```

Comment:
Same here.

Reply:
Yes, naturally, I copied the same code in a few places. I'll address it everywhere.

Reply:
Yes, naturally, I copied the same code in a few places. I'll address it everywhere.